### PR TITLE
[D1] clarify help string for `-v2` flag

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**
Trivial

### CODE REVIEW

**Comment by mihails-strasuns-sociomantic** [comment]
ping @AndrejMitrovic

File: src/mars.c [hunk 1]
```diff
@@ -405,7 +405,7 @@ Usage:\n\
   -unittest      compile in unit tests\n\
   -v             verbose\n\
   -v1            D language version 1\n\
-  -v2            give hints for converting to D2 (default hint set)\n\
+  -v2            give hints for converting to D2 (enables all hints)\n\
   -v2=XXX        enable the specific XXX conversion hint type\n\
   -v2=-XXX       disable the specific XXX conversion hint type\n\
                  (if -v2 wasn't enable already, the defaults are enabled first)\n\
```
Comment (mathias-baumann-sociomantic) [inline src/mars.c]: Also references "defaults" here but doesn't specify them

**Comment by mihails-strasuns-sociomantic** [comment]
Updated

**Comment by mathias-baumann-sociomantic** [comment]
LGTM

**Pull request merged**